packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -652,9 +654,9 @@ export class AngularCompilerPlugin implements Tapable {
       // If we have a locale, auto import the locale data file.
       // This transform must go before replaceBootstrap because it looks for the entry module
       // import, which will be replaced.
-      if (this._compilerOptions.i18nInLocale) {
+      if (this._compilerOptions._normalizedLocale) {
```

Comment:
Is `_normalizedLocale` a new option in `AngularCompilerOptions`?

Reply:
Seems like he’s reading from the instance but setting in the options.

Reply:
Surprised that this passes build.

Reply:
sorry, my mistake, fixing this!